go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/zencoder/go-dash/v3
 
-go 1.16
```

Comment:
Maybe just leave at 1.16...or no greater than 1.18 (for bakery compatibility).

Reply:
back to 1.16